### ISSUE TITLE
New donation gif?

### ISSUE THREAD

**Issue by jbubar** [hackforla/website#1146]
### Overview
The current donation gif has some text to the right of it. I thought might want a new one.
here is the [link the gif live on the events page of our site](https://www.hackforla.org/about#donations)

<details >
<summary>current gif</summary>

![image](https://www.hackforla.org/assets/images/about/HfLA-Donate.gif)

</details>
<details>
<summary>new gif option</summary>

![image](https://user-images.githubusercontent.com/53061723/109564629-2e92fc80-7a96-11eb-8583-62aa980e8046.gif)

</details>

### Action Items
- [ ] Look through these gif options
- [ ] Get product's or design's thought on this
- [ ] create a new issue to replace it if we chose to do that

### Resources/Instructions
REPLACE THIS TEXT -If there is a website which has documentation that helps with this issue provide the link(s) here.

**Comment by akibrhast** [comment]
@daniellex0 Should we replace it with the new gif provided by @jbubar ?

**Comment by daniellex0** [comment]
@akibrhast Sure, and it looks like the donation form has a new section about repeated donations so this is great to update- thanks for doing that @jbubar ! 

@ExperimentsInHonesty just confirming this is fine to replace?

The only slight concern is that since this is a narrower image it might affect the formatting, but we can make adjustments to make sure everything is centered and looks good. FYI this gif currently exists in both the About Us page and the upcoming Join Us page.

**Comment by Oluwafemi-Jegede** [comment]
Approved by @ExperimentsInHonesty

**Comment by Lol-Whut** [comment]
I'm going to switch out the gifs and close the issue when completed

**Comment by Lol-Whut** [comment]
Progress: On my end, the gif is replaced and seems to be working, but I'm having trouble pushing changes; I'm thinking this is a git flow issue I'm encountering for the first time.
Blockers: I've made changes on my feature branch, but when I use the "git status" command, I get a "working tree clean" message. I've tried different methods of making these changes to see if my original method was the problem; at this point I would expect the HFLA-Donations.gif file and the about.html file that links to it to show up with changes to commit. 
Availability: 4-5 hours
ETA: If I can figure out the issue, pushing changes will take a couple mins; hopefully researching the issue won't take tooo much more time.
Pictures: Current view of website on my end -- 

![image](https://user-images.githubusercontent.com/40847248/111075083-3a2dec80-84b4-11eb-9d74-86ac7ed74fa2.png)
